src/views/webview-app/components/connect-form/ssl-tab/ssl-server-client-validation.tsx

```diff
@@ -64,15 +62,15 @@ class SSLServerClientValidation extends React.Component<props> {
         className={classnames(styles['form-group'])}
       >
         <FileInputButton
-          label="Certificate Authority"
+          label="Certificate Authority (.pem)"
```

Comment:
👍

Reply:
Hmm good thought on resetting the fields. We can chat w/ Claudia on that.